### ISSUE TITLE
Unable to deploy webapp into WWW

### ISSUE THREAD

**Issue by LogeswaranAudhikesavan** [Altoros/fabric-rest#3]
Hi Team -
I tried to place my webapp into www(static file) index.html. It is not showing up once I deployed & started the server(fabric-starter). 

Any other steps to be done, once we place the static file in www?. like middleware to be updated?, please share the instructions.

Any open issue?

thanks

**Comment by MaxXx1313** [comment]
Hi
Have you resolved the issue?

**Comment by LogeswaranAudhikesavan** [comment]
Nope, I have created separate Docker image to use my application, but didn't go successful. :(

**Comment by MaxXx1313** [comment]
In case you are using docker image, you should map docker volume to `www` folder:
```bash
docker run -v /path/to/your/www:/usr/src/app/www maxxx1313/fabric-rest
```

Otherwise you can replace `server/www` folder content (initially there is an [index.html](https://github.com/Altoros/fabric-rest/tree/master/server/www) file)

**Comment by LogeswaranAudhikesavan** [comment]
Sure, will give a try and update!. thanks for taking your time to revert back!!

**Comment by push1st1k** [comment]
Hi @LogeswaranAudhikesavan! 
Did you manage to solve your issue?

**Comment by LogeswaranAudhikesavan** [comment]
Nope!, I am starting fresh.. I couldn't map docker volume. Are you facing the same issue?

thanks
Lokesh
